pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -1,6 +1,21 @@
-{ stdenv, fetchurl, pkgs, lib }:
+{ mkDerivation, stdenv, fetchurl, pkgs, lib, qtbase }:
+let
 
-stdenv.mkDerivation rec {
+  ldLibraryPath = with pkgs; stdenv.lib.makeLibraryPath  [
+    libbsd.out
+    libffi.out
+    gmpxx.out
+    python27Full.out
+    python27Packages.libxml2.out
+    qt5.qtbase
+    zlib
+    xlibs.libX11.out
+    xorg_sys_opengl.out
+    xlibs.libXrender.out
+    gcc-unwrapped.lib
+  ];
+
+in mkDerivation rec {
```

Comment:
We are still doing this? https://github.com/NixOS/nixpkgs/issues/65399#issuecomment-519426193

Reply:
[Afaik, and according to the docs: yes](https://nixos.org/nixpkgs/manual/#sec-language-qt)

> Use mkDerivation instead of stdenv.mkDerivation. mkDerivation is a wrapper around stdenv.mkDerivation which applies some Qt-specific settings. This deriver accepts the same arguments as stdenv.mkDerivation; refer to Chapter 6, The Standard Environment for details.
>
> To use another deriver instead of stdenv.mkDerivation, use mkDerivationWith:
> 
>```
>mkDerivationWith myDeriver {
>  # ...
>}
>```

Reply:
Hmm, https://github.com/NixOS/nixpkgs/pull/71089 is still not merged. I would suggest following it anyway so that we do not have to change it back soon  after.

Reply:
done, i think...